app/controllers/users/emails_controller.rb

```diff
@@ -62,6 +63,10 @@ def delete
       redirect_to account_url
     end
 
+    def pending_completions_consent?
+      needs_completion_screen_reason
```

Comment:
A method suffixed with `?` should return a boolean value.

I think we can use `.present?` to check whether a reason exists and return a boolean.

```suggestion
    def pending_completions_consent?
      needs_completion_screen_reason.present?
```

Reply:
Changed to that and updated the spec.